showcase/korogenoest.md

```diff
@@ -0,0 +1,130 @@
+
+## Considerations in Estonia
+
+- Estonia has a single national Health Board
+- e-Estonia and the X-road (e-Identity, e-Health Records, e-Governance) infrastructure allows easy data linking across data sources using the personal identification number.
+- There has been previous studies and cooperation between the Health Board and the University of Tartu in virus monitoring (HIV, Flu)
+- There were no additional local laws restraining the study
+- While the code of the dashboards, data handling is not publicly available the team is happy to consult researchers facing similar situation in the future. The best way is to send an email to [ELIXIR-Estonia](elixir@ut.ee).  The original [Galaxy workflows](https://workflowhub.eu/workflows?filter%5Bproject%5D=33&filter%5Bquery%5D=covid) adapted and used are publicly available. 
+
+## External links to resources relevant to the KoroGeno-Est showcase
+
+- [Estonian Covid19Dataportal](https://covid19dataportal.ee/en/) 
+- [Overview of the SARS-CoV-2 projects (in Estonian)](https://kliinilinemeditsiin.ut.ee/et/sisu/eesti-sars-cov-2-taisgenoomide-jarjestamine-ja-analuus-korogeno-est-1est-2est-3) 
+[- Information on how to register a study to ENA](https://ena-docs.readthedocs.io/en/latest/submit/study.html)
+- [ENA virus pathogen reporting standard checklist](https://www.ebi.ac.uk/ena/browser/view/ERC000033) 
+- [ENA metadata templates](https://github.com/ELIXIR-Belgium/ENA-metadata-templates) 
+- [Galaxy wrapper for the ENA upload tool](https://github.com/usegalaxy-eu/ena-upload-cli) 
+- [Submission of SARS-Cov-2 raw reads to ENA in Galaxy](https://rdm.elixir-belgium.org/covid-19/sarscov2_submission.html)
+- [General Guide On ENA Data Submission by ENA](https://ena-docs.readthedocs.io/en/latest/submit/general-guide.html)
+- [European Nucleotide Archive Quick tour by EMBL-EBI](https://www.ebi.ac.uk/training/online/courses/ena-quick-tour/)
+- [Example metadata template ERC000033](https://github.com/ELIXIR-Belgium/ENA-metadata-templates/blob/main/templates/ERC000033/example_metadata_template_ERC000033.xlsx )
+
+## Team members of KoroGeno-Est
+Core team at University of Tartu:
+Virology and epidemiology: Kristi Huik (main coordinator of the project); Aare Abroi, Irja Lutsar, Radko Avi, Merit Pauskar, Taavi Päll, Arina Shablinskaja, Kai Truusalu, Dagmar Hoidmets, Ene-Ly Jõgeda, Eveli Kallas, Katrin Kaarna (project management)
+Sequencing: Tuuli Reisberg (coordinator of sequencing), Mait Metspalu, Lili Milani, Steven Smit
+Bioinformatics: Hedi Peterson (coordinator of bioinformatic), Ulvi Gerst Talas, Ivar Koppel, Ott Eric Oopkaup, Erik Jaaniso, Ivan Kuzmin, Uku Raudvere; Heleri Inno, Diana Pilvar (data managers) 
+
+Synlab team: Andrio Lahesaare, Paul Naaber, Krislin Raus, Kaspar Ratnik
+
+Estonian Health Board team: Mari-Anne Härma, Heiki Niglas, Liidia Dotsenko, Olga Sadikova, Hanna Sepp, Heleene Suija, Jevgenia Epštein 
+
```

Comment:
```suggestion

### Sponsors
The KoroGeno-EST studies were conducted in cooperation with the University of Tartu, the Estonian Health Board and SYNLAB Eesti OÜ during 2021-2022. The [KoroGeno-EST-3 and KoroGeno-EST-2022 projects](https://kliinilinemeditsiin.ut.ee/et/sisu/eesti-sars-cov-2-taisgenoomide-jarjestamine-ja-analuus-korogeno-est-1est-2est-3) were financed by the Republic of Estonia's Ministry of Education and Research.
```

Reply:
Sponsors is not applicable and then the first sentence reads also odd as it describes the collaborations. I put funding information as the subtitle but this needs some editorial discussion